src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ParseJson.txt

```diff
@@ -224,3 +224,7 @@ Blank()
 
 >> Text(Index(ParseJSON("[ { ""a"" : 10 }, { ""b"" : ""str"" } ]"),2).a)
 Blank()
+
+// ForAll
+>> ForAll(ParseJSON("[5, 6, 7]"), Value(Value))
```

Comment:
>Value [](http://example.com/codeflow?start=40&length=5)

Should we wrap the value with the {Value} record at this point, or let the user access the item directly with `ThisRecord`?

Reply:
You are correct, Greg wants ThisItem to be used. I will fix this problem in the next commit.

Reply:
Changed to ThisItem. This required adding a new scope type.